guess-the-country/country-frontend/src/components/Notification.vue

```diff
@@ -53,12 +48,38 @@ export default {
         type: String
       },
   },
+  computed: {
+    getQuestionCondition() {
+      let question_condition = this.sequence_mode==='classic'&&!this.user_city_answer
+                  || this.sequence_mode==='recommender'&&!this.prediction_city
+                  || this.sequence_mode==='basic'&&!this.user_city_answer
+      return question_condition
+    },
+    getUserGuessCondition() {
+      let user_guess_condition = this.sequence_mode==='classic'&&this.user_city_answer && !this.prediction_city
+                  || this.sequence_mode==='recommender'&&this.user_city_answer
+                  || this.sequence_mode==='basic' && this.user_city_answer
+      return user_guess_condition
+    },
+    getAIGuessCondition_control() {
+      let ai_guess_condition_control = this.sequence_mode==='classic'&&this.prediction_city && this.control
+                  || this.sequence_mode==='recommender'&& this.prediction_city&&!this.user_city_answer && this.control
+      return ai_guess_condition_control
+    },
+    getAIGuessCondition_treatment() {
+      let ai_guess_condition_treatment = this.sequence_mode ==='classic' && this.prediction_city && !this.control
+                  || this.sequence_mode==='recommender' &&this.prediction_city&&!this.user_city_answer && !this.control
+      return ai_guess_condition_treatment
+    },
```

Comment:
Gleiche Anmerkungen wie oben :-)

Rein semantisch ist "computed" genauso wie "data" eine Variable, d.h. das "get" davor ist nicht sinnvoll. Einfach genau so als Wert benennen wie die props oder die Daten in "data".